app/assets/stylesheets/bulk-actions.scss

```diff
@@ -4,14 +4,21 @@
     display: flex;
     flex-direction: row;
     flex: 1 0 auto;
+    flex-wrap: wrap;
     align-items: center;
     .last-element {
       margin-left: auto;
       display: flex;
-      flex-direction: row-reverse;
+      //flex-direction: row-reverse;
```

Comment:
Can we remove the commented code?

Reply:
Will do after I finish the assessment dropdown.